variables.tf

```diff
@@ -142,3 +142,28 @@ variable "access_tags" {
   description = "A list of access tags to apply to the Key Protect instance created by the module. Only used if 'create_key_protect_instance' is true."
   default     = []
 }
+
+##############################################################
+# Context-based restriction (CBR)
+##############################################################
+
+variable "cbr_rules" {
+  type = list(object({
+    description = string
+    account_id  = string
+    rule_contexts = list(object({
+      attributes = optional(list(object({
+        name  = string
+        value = string
+    }))) }))
+    enforcement_mode = string
+    operations = optional(list(object({
+      api_types = list(object({
+        api_type_id = string
+      }))
+    })))
+  }))
+  description = "(Optional, list) List of context-based restrictions rules to create"
```

Comment:
Add a note outlining that this only applies to Key Protect instances and not HPCS instances

Reply:
No need to add `(Optional, list)`. Also you should make it clear in the description that its only applicable when create a new, or passing an existing Key Protect instance, and that its not supported fro HPCS

Reply:
@dishankkalra23 I think you missed this comment?